README.md

```diff
@@ -133,6 +133,72 @@ For more details on job flow within our architecture see our [**swimlane chart**
   
     ```
 
+### Hosted cluster provisioning example: _(KubeVirt)_
+
+  * When creating the `HostedCluster` and `NodePool` resource add the `spec.pausedUntil` field with value `true` to both resources. If using the HostedCluster CLI you can specify the flag `--pausedUntil true`.
```

Comment:
If using the HostedCluster CLI --> If using the `hcp create cluster` CLI

Reply:
Done. Thanks!